### PR TITLE
add new Voice system

### PR DESCRIPTION
**Pull request by RichsJeson**
Add a new voice system, Whale AI Voice.

### CODE REVIEW

**Comment by RichsJeson** [comment]
Add a new voice system, Whale AI Voice.

**Comment by RichsJeson** [comment]
update zeasn  asr description

**Comment by RichsJeson** [comment]
Added an entry for Zeasn ASR, and made associated formatting changes, to the app registry table

File: DAB.md [hunk 1]
```diff
@@ -1937,6 +1938,7 @@ versions | a list of versions of the protocol implemented on the device
 | LG ThinQ           |   LGThinQ               |
 | Yandex Alice       |   YandexAlice           |
 | TiVo Conversation  |   TiVoConversation      |
+| Whale  AI Voice    |   Zeasn ASR             |
```
Comment (bgy36ww) [inline DAB.md:1941]: Is it possible to not have a space here?

**Review by bgy36ww — CHANGES_REQUESTED** [review]

File: DAB.md (hunk 1 above)
Comment (RichsJeson) [reply DAB.md:1941]: Sorry, I didn't understand what you mean. Can you explain it in more detail?

Comment (RichsJeson) [reply DAB.md:1941]: At present, we are still making adjustments.

**Comment by RichsJeson** [comment]
Hey

Wei Wang ***@***.***> 于2024年8月29日周四 04:16写道：

**Pull request closed without merging**